index.js

```diff
@@ -436,7 +429,7 @@ Router.prototype.findRoute = function findNode (method, path, constraints = {})
       return {
         handler: existRoute.handler,
         store: existRoute.store,
-        params: existRoute.params || []
```

Comment:
Uncovered branch.

`route.params` is assigned as an array and never reassigned, do I miss something?